.github/workflows/ci.yml

```diff
@@ -21,15 +21,14 @@ jobs:
         python-version: ["3.9", "3.13"]
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
```

Comment:
I moved python-version down to astral setup. But if you still want setup-python it is ok with me.

Reply:
that’s fine!